apps/notifier/src/assets/views/hola.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hola!</title>
+</head>
+<body>
+    <h1>Encantado de conocerte</h1>
```

Comment:
I'm also pleased to meet you, but we should think about a proper template to welcome our users please 😂